src/magicbox/ResultPreviewsGrid.ts

```diff
@@ -0,0 +1,398 @@
+import { $$, Dom } from '../utils/Dom';
+import _ = require('underscore');
+import { Assert } from '../misc/Assert';
+import { defaults } from 'underscore';
+import { Component } from '../ui/Base/Component';
+
+enum Direction {
+  Up = 'Up',
+  Down = 'Down',
+  Left = 'Left',
+  Right = 'Right'
+}
+
+export interface SearchResultPreview {
+  dom: HTMLElement;
```

Comment:
Confusing a bit that dom is an HTMLElement, dom is a class  `Dom` so I would expect the type to be `Dom` or the name to be `element` or `previewElement`

Reply:
Good point, I'll change it to `element`.